### PR TITLE
Tup #29 add profile component

### PR DESCRIPTION
**Pull request by JohnXander**

### CODE REVIEW

File: src/components/users/registration/RegistrationPage.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const RegistrationPage = () => {
   const registerUser = (event) => {
     event.preventDefault()
     client.post('/user', user, false)
-      .then(res => setRegisterResponse(res.data))
+      .then(res => setRegisterResponse(res.data), console.log("hi", registerResponse))
```
Comment (Auenc) [inline src/components/users/registration/RegistrationPage.js:15]: Do we need this console.log now?

If not, it's always best to remove them so we can keep the console clean.

**Review by Auenc — CHANGES_REQUESTED** [review]
Just a quick question on whether that `console.log` is needed.

File: src/components/users/registration/RegistrationPage.js (hunk 1 above)
Comment (JohnXander) [reply src/components/users/registration/RegistrationPage.js:15]: My bad. Didn't realise that was still there🤦

**Review by Auenc — APPROVED** [review]
Looks good to me!

**Review by sirpae — APPROVED** [review]

File: src/components/profile/Profile.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import Avatar from '@mui/material/Avatar';
+import Link from '@mui/material/Link';
+import './style.css'
+
+import Header from '../Header/Header';
+
+
+const Profile = ({ profileData }) => {
+    const { first_name, last_name, biography, github_url } = profileData
+    console.log(first_name)
```
Comment (Auenc) [inline src/components/profile/Profile.js:11]: Can we remove this console.log please?

**Review by Auenc — APPROVED** [review]
Looks good to me

File: src/components/profile/Profile.js (hunk 2 above)
Comment (JohnXander) [reply src/components/profile/Profile.js]: Done 👍

**Review by sirpae — APPROVED** [review]

**Review by vherus — APPROVED** [review]

**Pull request merged**